macro-analytics/Code.js

```diff
@@ -0,0 +1,560 @@
+    .setTransposeRowsAndColumns(false)
+    .setNumHeaders(1)
+    .setHiddenDimensionStrategy(Charts.ChartHiddenDimensionStrategy.IGNORE_BOTH)
+    .setOption('bubble.stroke', '#000000')
+    .setOption('useFirstColumnAsDomain', true)
+    .setOption('pieHole', 0.5)
+    .setOption('isStacked', 'false')
+    .setOption('title', 'Page views by country this year')
+    .setOption('annotations.domain.textStyle.color', '#808080')
+    .setOption('textStyle.color', '#000000')
+    .setOption('legend.textStyle.color', '#191919')
+    .setOption('pieSliceTextStyle.color', '#000000')
+    .setOption('titleTextStyle.color', '#757575')
+    .setOption('annotations.total.textStyle.color', '#808080')
+    .setPosition(27, 4, 9, 19)
+    .build();
+    sheet.insertChart(chart);
+    charts = sheet.getCharts();
+    chart = charts[charts.length - 1];
+    sheet.removeChart(chart);
+    chart = sheet.newChart()
+    .asPieChart()
+    .addRange(spreadsheet.getRange('B28:C60'))
+    .setMergeStrategy(Charts.ChartMergeStrategy.MERGE_COLUMNS)
+    .setTransposeRowsAndColumns(false)
+    .setNumHeaders(1)
+    .setHiddenDimensionStrategy(Charts.ChartHiddenDimensionStrategy.IGNORE_BOTH)
+    .setOption('bubble.stroke', '#000000')
+    .setOption('useFirstColumnAsDomain', true)
+    .setOption('legend.position', 'top')
+    .setOption('pieHole', 0.5)
+    .setOption('isStacked', 'false')
+    .setOption('title', 'Page views by country this year')
+    .setOption('annotations.domain.textStyle.color', '#808080')
+    .setOption('textStyle.color', '#000000')
+    .setOption('legend.textStyle.color', '#191919')
+    .setOption('pieSliceTextStyle.color', '#000000')
+    .setOption('titleTextStyle.color', '#757575')
+    .setOption('annotations.total.textStyle.color', '#808080')
+    .setPosition(27, 4, 9, 19)
+    .build();
+    sheet.insertChart(chart);
+    charts = sheet.getCharts();
+    chart = charts[charts.length - 1];
+    sheet.removeChart(chart);
+    chart = sheet.newChart()
+    .asPieChart()
+    .addRange(spreadsheet.getRange('B28:C60'))
+    .setMergeStrategy(Charts.ChartMergeStrategy.MERGE_COLUMNS)
+    .setTransposeRowsAndColumns(false)
+    .setNumHeaders(1)
+    .setHiddenDimensionStrategy(Charts.ChartHiddenDimensionStrategy.IGNORE_BOTH)
+    .setOption('bubble.stroke', '#000000')
+    .setOption('useFirstColumnAsDomain', true)
+    .setOption('legend.position', 'labeled')
+    .setOption('pieHole', 0.5)
+    .setOption('isStacked', 'false')
+    .setOption('title', 'Page views by country this year')
+    .setOption('annotations.domain.textStyle.color', '#808080')
+    .setOption('textStyle.color', '#000000')
+    .setOption('legend.textStyle.color', '#191919')
+    .setOption('pieSliceTextStyle.color', '#000000')
+    .setOption('titleTextStyle.color', '#757575')
+    .setOption('annotations.total.textStyle.color', '#808080')
+    .setPosition(27, 4, 9, 19)
+    .build();
+    sheet.insertChart(chart);
+  };
```

Comment:
Please add a single blank line at the end of all files

Reply:
Good call, added!